app/src/main/java/com/keylesspalace/tusky/adapter/TagViewHolder.kt

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2023 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.adapter
+
+import android.view.View
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.entity.TrendingTagHistory
+import com.keylesspalace.tusky.interfaces.LinkListener
+import com.keylesspalace.tusky.view.GraphView
+import com.keylesspalace.tusky.viewdata.TrendingViewData
+
+class TagViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
```

Comment:
`TrendingTagViewHolder`? To avoid confusion with other view holders that handle hashtags?

Reply:
+1 Resolved locally.